staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/inner/validatefalse/doc_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inner
+
+import (
+	"testing"
+
+	operation "k8s.io/apimachinery/pkg/api/operation"
+)
+
+func TestInnerValidationWithValidateFalse(t *testing.T) {
```

Comment:
If you use the same scaffolding as other tests and run with `UPDATE_VALIDATION_GEN_FIXTURE_DATA=true` you get a nice test for free.

Reply:
Ah thanks!  I've added the below lines to the the validatefalse and metadataname doc.go files now:
```
// +k8s:validation-gen-test-fixture=validateFalse

...
// +k8s:validateFalse="type T1"
type T1 struct {
...
```

I am seeing the generated test fixtures now, ex:
`staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/inner/validatefalse/testdata/validate-false.json`
```
{
    "*inner.T1": {
      "": [
        "type T1"
      ],
      "t2.mapField": [
        "inner T1.T2.MapField"
      ],
      "t2.pointerField": [
        "inner T1.T2.PointerField"
      ],
      "t2.sliceField": [
        "inner T1.T2.SliceField"
      ],
      "t2.stringField": [
        "inner T1.T2.StringField"
      ]
    }
  }
```